### DISCUSSION TITLE
add switch optionsI, Infinite String Farm

### DISCUSSION THREAD

**Discussion by CatTeaA** [PurpurMC/Purpur#1368] [__Dismissed__] [answered]
the method of using water to break a Tripwire that is disabled by using Shears to infinitely duplicate String is still valid. I believe this is a serious bug
I asked the paperMC team before but didn't adopt it

Link to relevant video：
https://forums.papermc.io/threads/vulnerabilities-remain-still-replicable-infinite-string-farm-in-paper-1-19-4.718/

Infinite String Farm,Whether you can add a switch option

**Comment by Huge-mistake** [comment]
![2023-06-27_15 36 46(1)(1)](https://github.com/PurpurMC/Purpur/assets/137968615/c94fdf6f-f894-4261-8bdf-3a6e703c59b5)

![Screenshot_2023-06-28-09-07-31-542_tv danmaku bili](https://github.com/PurpurMC/Purpur/assets/137968615/4e0658ec-a0cb-4dd3-b9dc-a4b23b6793b8)
https://b23.tv/gUA0091

Tens of millions of efficiencies can be obtained through stacking        2850k/h (String)    
Players can craft tens of millions of wool
Hope to add option to fix

**Comment by Huge-mistake** [comment]
I also don't want the server to have any vulnerabilities to copy items

**Reply by Huge-mistake** [reply]
@BillyGalbreath 
I need your help

**Comment by BillyGalbreath** [comment, marked as answer]
I looked through the server code a bit to try and figure out what's wrong here, but I failed.

However, I was able to write [this small plugin](https://github.com/BillyGalbreath/TripwireFix/releases/tag/v1.0) that fixes the issue. Just install it in your plugins directory like any other plugin. There's no commands or permissions. It just works.

**Reply by Walkersifolia** [reply]
Can purpur #build 2006 use this plugin without any settings?
